### PR TITLE
ISSUE-217: Ensures info['uid'] to be present for both processing options

### PR DESCRIPTION
**Pull request by DiegoPino**
See #217

### CODE REVIEW

**Comment by DiegoPino** [comment]
@patdunlavey can you please give this a check?

File: src/Form/amiSetEntityDeleteProcessedForm.php [hunk 1]
```diff
@@ -85,6 +85,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     }
     if ($file && $data!== new \stdClass()) {
       // Only UUIDs you can delete will be added.
+      $data->info['uid'] = $this->currentUser()->id();
```
Comment (patdunlavey) [inline src/Form/amiSetEntityDeleteProcessedForm.php:88]: The ami set has a uid value - the author of the set. I'm not sure which uid makes more sense: the author of the ami set, or the user currently processing the set?

Comment (DiegoPino) [reply src/Form/amiSetEntityDeleteProcessedForm.php:88]: The user processing in my opinion ... At least that has been the behavior. If we want to change that we can introduce a new feature/option in the future.

Comment (patdunlavey) [reply src/Form/amiSetEntityDeleteProcessedForm.php:88]: That works for me - just wanted to make sure it was considered!

I have an AMI set processing (successfully - no white screen) on my local now. After that, I'll test deleting processed ADOs.

Comment (patdunlavey) [reply src/Form/amiSetEntityDeleteProcessedForm.php:88]: @DiegoPino  CONFIRMED: both processing and deleting processed ADOs are successful!

**Review by patdunlavey — APPROVED** [review]
Confirmed in local testing

File: src/Form/amiSetEntityDeleteProcessedForm.php (hunk 1 above)
Comment (patdunlavey) [reply src/Form/amiSetEntityDeleteProcessedForm.php:88]: I also checked that enqueued csv expansion and ado processing still work, and they do.

**Comment by patdunlavey** [comment]
Do you need a PR to get this merged to 0.9.0?

**Pull request merged**